static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -27,28 +33,43 @@ export default class FinancialAidCard extends React.Component {
     program: Program,
     coursePrice: CoursePrice,
     openFinancialAidCalculator: () => void,
-    documentSentDate: Object,
-    setDocumentSentDate: Function,
+    documents: DocumentsState,
+    setDocumentSentDate: (sentDate: string) => void,
+    updateDocumentSentDate: (financialAidId: number, sentDate: string) => Promise<*>,
+    fetchDashboard: () => void,
+  };
+
+  submitDocuments = (): void => {
+    const {
+      fetchDashboard,
+      program,
+      documents,
+      updateDocumentSentDate,
+    } = this.props;
+    const financialAidId = program.financial_aid_user_info.id;
+
+    updateDocumentSentDate(financialAidId, documents.documentSentDate).then(() => {
+      fetchDashboard();
+    });
   };
 
   renderDocumentStatus() {
     const {
       setDocumentSentDate,
-      documentSentDate,
+      documents,
     } = this.props;
-    const { application_status: applicationStatus } = this.props.program.financial_aid_user_info;
-
-    let errors = {};
-    if (documentSentDate.edit !== undefined) {
-      errors = documentSentDate.edit.errors;
-    }
+    const {
+      application_status: applicationStatus,
+      date_documents_sent: dateDocumentsSent,
+    } = this.props.program.financial_aid_user_info;
```

Comment:
are you not putting this in the destructuring above just to avoid semi-ridiculous nesting?